nemo/src/io/parser/types.rs

```diff
@@ -279,12 +279,22 @@ impl FromExternalError<Span<'_>, ParseError> for LocatedParseError {
 
 impl FromExternalError<Span<'_>, ParseIntError> for LocatedParseError {
     fn from_external_error(input: Span, _kind: ErrorKind, e: ParseIntError) -> Self {
-        ParseError::ExternalError(Box::new(e.into())).at(input)
+        ParseError::ExternalError(Box::new(crate::error::Error::PhysicalError(e.into()))).at(input)
     }
 }
 
 impl FromExternalError<Span<'_>, ParseFloatError> for LocatedParseError {
     fn from_external_error(input: Span, _kind: ErrorKind, e: ParseFloatError) -> Self {
```

Comment:
I'm not too happy about `ParseIntError` and `ParseFloatError` being in the physical layer, but I guess they need to be there … :/

Reply:
Just to clarify, this is not a blocker, but it will make some parse errors more difficult to read. Another option might be to duplicate these two in both error types.